cmake/nvml-config.cmake

```diff
@@ -0,0 +1,35 @@
+include(FindPackageHandleStandardArgs)
+include("$ENV{WORKSPACE_TOP}/common/cmake/lb.cmake")
+
+lb_local_path(PMEM-SDK_BUILD nvml pmem-sdk)
+
+find_path(Pmem-sdk_INCLUDE_DIR
+  "libpmem.h"
+  PATHS "$ENV{WORKSPACE_TOP}/nvml"
+  PATH_SUFFIXES "src/include"
+)
+
+find_library(Pmem-sdk_LIBRARY NAMES libpmem.a PATHS "${PMEM-SDK_BUILD}/lib")
+
+lb_version(PMEM-SDK_VERSION nvml pmem-sdk)
+
+find_package_handle_standard_args(Pmem-sdk DEFAULT_MSG
```

Comment:
seems OK but any reason not to use pmem shipped in ubuntu 18.04 ?

Reply:
@maor-lb Good point, I'll check with libpmem-dev.
Still I'd merge this so our nvml would be available.